ghci/containers/ghci-koji/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM registry.fedoraproject.org/fedora:32
+
+# koji db schema is in docs, remove nodocs from from dnf config
+RUN sed  -i '/^tsflags=nodocs$/d' /etc/dnf/dnf.conf \
+    && dnf \
+            -y \
+            --setopt=fastestmirror=True \
+            --setopt=install_weak_deps=False \
+            install koji-hub postgresql \
+    && dnf clean all
+
+RUN mkdir -p /mnt/koji/{packages,repos,work,scratch,repos-dist} \
+    && chown -R apache:apache /mnt/koji
```

Comment:
I think you can split this into 2 `RUN` lines. These line-breaks are usually less-readable, I think.

Reply:
yep